### PR TITLE
instructions for open climate data extraction

### PR DESCRIPTION
**Pull request by Liyubov**
many people asked me how to download or extract climate data, here i tried to describe how to do it with data for UK for temperature from https://www.ceda.ac.uk/ website

issues with these data: climate data are claimed to be open, but in fact one needs to learn how to extract them.
hopefully it will be helpful.

### CODE REVIEW

**Comment by rufuspollock** [comment]
@Liyubov this is awesome - thanks for sharing this. Can you check you are happy for this to be openly licensed (e.g. MIT or Public Domain).

@lauragift21 I think we can merge as is once we have checked re license and then you could do any grammar / formatting improvements that are needed 😄

File: climate-data.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+# Climate open data
```
Comment (lauragift21) [inline climate-data.md]: This looks great! I've added a few grammar/formatting suggestions.

File: climate-data.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Everyone can have access to climate data, however many people were not able to access the data due to various reasons: failing register on the website, 
+formulating and sending request to get the right data they need.
```
Comment (lauragift21) [inline climate-data.md]: Anyone can have access to climate data, however, it is difficult to gain access to the data due to various reasons:
- failing registration on the website
- formulating and sending a request to get the right data they need

File: climate-data.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+Here we write precise
```
Comment (lauragift21) [inline climate-data.md]: I'm not sure if this sentence fits in.

File: climate-data.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+formulating and sending request to get the right data they need.
+Here we write precise 
+
+We use several sources for open data on temperature:
+1. reanalysis data from GES DISC MERRA from https://disc.gsfc.nasa.gov/
+2. observational data: example data available for downloading after login http://data.ceda.ac.uk/badc/ukmo-hadobs/data/insitu/MOHC/HadOBS/HadUK-Grid/v1.0.0.0/60km/  and downloadable data link url = 'http://dap.ceda.ac.uk/thredds/fileServer/badc/ukmo-hadobs/data/insitu/MOHC/HadOBS/HadUK-Grid/v1.0.0.0/60km/tas/mon/v20181126/tas_hadukgrid_uk_60km_mon_188401-188412.nc' and https://services.ceda.ac.uk/cedasite/myceda/user/ 
+
+# How to download the data 
+1. Create account and log in to https://www.ceda.ac.uk/ 
+2. Depending on the variables and time-scale of the data you want to download setup request, e.g. for 2010 taxmax (temperature): http://dap.ceda.ac.uk/thredds/fileServer/badc/ukmo-hadobs/data/insitu/MOHC/HadOBS/HadUK-Grid/v1.0.0.0/60km/tas/mon/v20181126/tas_hadukgrid_uk_60km_mon_201001-201012.nc
```
Comment (lauragift21) [inline climate-data.md]:
```suggestion
2. Depending on the variables and time-scale of the data you want to download setup request, for example, 2010 taxmax (temperature): http://dap.ceda.ac.uk/thredds/fileServer/badc/ukmo-hadobs/data/insitu/MOHC/HadOBS/HadUK-Grid/v1.0.0.0/60km/tas/mon/v20181126/tas_hadukgrid_uk_60km_mon_201001-201012.nc
```

Comment (Liyubov) [reply climate-data.md]: thank you for suggestions. i agree with them. i also provide information about the data about public and restricted data from ceda datasets https://help.ceda.ac.uk/article/98-accessing-data

File: climate-data.md (hunk 3 above)
Comment (Liyubov) [reply climate-data.md]: indeed i suggest to remove it and just add that "Detailed guide for searching and downloading climate data is sometimes lacking. "

**Comment by Liyubov** [comment]
Hi, @lauragift21 @rufuspollock 
Thank you for changes and suggestions. 

**About license on the data:** 
Data provided by CEDA are in the CF-netCDF format distributed via an Open Government Licence (OGL). More information about it is here: https://help.ceda.ac.uk/article/235-climate-data-in-ceda-archives
OGL permits anyone to copy, publish, distribute, transmit and adapt the licensed work, and to exploit it both commercially and non-commercially. In return, the re-user of the licensed work has to acknowledge the source of the work and (if possible) provide a link to the OGL.

**Code for parsing the data in netCDF** is here https://github.com/Liyubov/climate_open_data/blob/master/parse_hadukgrid.py 
Thank you!

**Comment by rufuspollock** [comment]
@lauragift21 can you go ahead and merge this 😄

File: climate-data.md (hunk 2 above)
Comment (Liyubov) [reply climate-data.md]: great, thank you. i agree with the changes!

**Pull request merged**